WalletWasabi.Fluent/Views/Wallets/Home/History/Details/TransactionDetailsView.axaml

```diff
@@ -27,7 +27,10 @@
                      Label="Amount"
                      TextValue="{Binding Amount}">
         <c:PrivacyContentControl>
-          <TextBlock Text="{Binding Amount, StringFormat={}{0} BTC, Mode=OneWay}" Classes="monoSpaced" />
+          <StackPanel Orientation="Horizontal">
+            <TextBlock Text="-" IsVisible="{Binding IsOutgoingAmount}" Classes="monoSpaced" />
+            <TextBlock Text="{Binding Amount, StringFormat={}{0} BTC, Mode=OneWay}" Classes="monoSpaced" />
+          </StackPanel>
```

Comment:
+1 point for creativity, but we should rather take one step towards a better transaction details page.

Reply:
Well if y'all are okay with that kind of change then I'll do that instead. I was trying to avoid any visual changes to the dialog that's why i got a little bit creative :laughing:

Reply:
Yepp, that change is fine, you can implement it.